internal/k3s/k3s.go

```diff
@@ -148,6 +148,14 @@ func StartCluster(verbose bool, timeout time.Duration, clusterName string, worke
 		cmdArgs = append(cmdArgs, srvArg)
 	}
 
+	//add further k3d args which are not offered by the Kyma CLI flags
+	for _, k3dArg := range k3dArgs {
+		if k3dArg == "" {
```

Comment:
Maybe it would be interesting to see what happens here if someone passes something like:
`--k3d-arg="     "`

Reply:
ah good catch.
It will then return an error to the user `FATA[0000] accepts between 0 and 1 arg(s), received 2 `.
We can do one of the 2 things:
1- either we even remove the current check `if k3dArg == ""` and let the `k3d` command return an error to the user whenever he passes an empty flag
2- or we could adapt our check and make it trim trailing and leading spaces and then check if the string is empty

Reply:
I was having the option 1 in mind, since it will transparently tell exactly what arguments are wrong.